packages/account-v2/src/modules/POAForm/POAFormContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { useActiveAccount } from '../../../../api/src/hooks';
+import { AUTH_STATUS_CODES, P2P_ROUTE } from '../../constants/constants';
+import { AddressDetailsForm as ProofOfAddressForm } from '../../containers/POAForm/AddressDetailsForm';
```

Comment:
**NIT**: Lets try to avoid aliasing components. It gets confusing to maintain and enhance

Reply:
Done.